src/client.js

```diff
@@ -320,6 +322,17 @@ export class Saturn {
     }
 
     if (lastError) {
+      const originUrl = opts.originURL ?? this.opts.originURL
```

Comment:
`originURL` is a bit ambiguous. Perhaps `fallbackURL`, or since the fallback term is already in use, `customerFallbackURL` or `lastFallbackURL`, `finalFallbackURL`, etc.